dev/AppNotifications/AppNotificationManager.cpp

```diff
@@ -177,31 +179,78 @@ namespace winrt::Microsoft::Windows::AppNotifications::implementation
 
     winrt::Windows::Foundation::IAsyncAction AppNotificationManager::RemoveByIdAsync(uint32_t notificationId)
     {
-        throw hresult_not_implemented();
+        THROW_HR_IF(E_INVALIDARG, notificationId == 0);
+
+        co_await winrt::resume_background();
+
+        std::wstring appId{ RetrieveNotificationAppId() };
+        ToastNotifications_RemoveToast(appId.c_str(), notificationId);
```

Comment:
LOG_IF_FAILED

Reply:
Done

Reply:
Following off-line discussion, none of the remove* APIs throw because a toast did not exist. They will throw is something goes wrong though.